### PR TITLE
[MOS-871] Loudspeaker icon not updated when audio routing has changed

### PR DESCRIPTION
**Pull request by lukasmas**
**Your checklist for this pull request**

Make sure that this PR:
- [ ] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

**Review by Lefucjusz — APPROVED** [review]

File: module-services/service-audio/AudioServiceAPI.cpp [hunk 1]
```diff
@@ -90,16 +90,18 @@ namespace AudioServiceAPI
...
+        // return true;
     }
 
-    bool SendEvent(sys::Service *serv, audio::EventType eType, audio::Event::DeviceState state)
+    void SendEvent(sys::Service *serv, audio::EventType eType, audio::Event::DeviceState state)
     {
         auto msg = std::make_shared<AudioEventRequest>(eType, state);
-        return serv->bus.sendUnicast(msg, service::name::audio);
+        serv->bus.sendMulticast(msg, sys::BusChannel::ServiceAudioNotifications);
+        // return true;
```
Comment (SP2FET) [inline module-services/service-audio/AudioServiceAPI.cpp:104]: why did you change it to void?

File: module-services/service-audio/ServiceAudio.cpp [hunk 2]
```diff
@@ -492,7 +497,7 @@ std::unique_ptr<AudioResponseMessage> ServiceAudio::HandleSendEvent(std::shared_
         input.audio->SendEvent(evt);
     }
 
-    return std::make_unique<AudioEventResponse>(RetCode::Success);
+    return sys::msgHandled();
```
Comment (SP2FET) [inline module-services/service-audio/ServiceAudio.cpp:500]: doesn't it break anything?

File: module-services/service-audio/AudioServiceAPI.cpp (hunk 1 above)
Comment (SP2FET) [reply module-services/service-audio/AudioServiceAPI.cpp:104]: nevermind - I didn't noticed the multicast

**Review by SP2FET — APPROVED** [review]

File: module-services/service-audio/AudioServiceAPI.cpp (hunk 1 above)
Comment (lukasmas) [reply module-services/service-audio/AudioServiceAPI.cpp:104]: Two reasons:
1. It is not checked anywhere what was the outcome of this function
2. Multicast doesn't return anything, so I cannot check if the message was actually sent or not

File: module-services/service-audio/ServiceAudio.cpp (hunk 2 above)
Comment (lukasmas) [reply module-services/service-audio/ServiceAudio.cpp:500]: seems not. 
This message is now multicast, so there is no need to send a response to a sender. Also, I've checked and multicast messages are handled like this.
